src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -214,7 +214,7 @@ static pbs_list_head pbs_vnode_set_list;      /* list of vnode set requests */
  * @param[in]	py_resource_str_value - a Python string object representing
  * 					the values in 'value_list'.
  * @param[in]	attr_def_p - the resource definition for the resource list
- * 			     represented by 'py_resource'.	
+ * 			     represented by 'py_resource'.
  * @param[in]	value_list - list of values cached for the 'py_resource' object
```

Comment:
I too don't see any harm, I feel that these do not fall as a part of this pull request.